doc/sphinx/addendum/type-classes.rst

```diff
@@ -289,8 +302,27 @@ declares each projection as an instance.
 .. warn:: Ignored instance declaration for “@ident”: “@term” is not a class
 
    Using this ``:>`` syntax with a right-hand-side that is not itself a Class
-   has no effect (apart from emitting this warning). In particular, is does not
-   declare a coercion.
+   has no effect (apart from emitting this warning).
+
+.. warn:: 'Class >' currently does nothing, a coercion will be introduced in future versions.
+
+   :g:`Class >` will behave as :g:`Record >` in future versions,
+   declaring the construcotr a coercion. Remove the :g:`>` to avoid
+   this warning.
```

Comment:
```suggestion
   While currently ignored, in a future version, :g:`Class >` will declare the constructor
   a coercion (consistent with :g:`Record >`). Remove the :g:`>` to avoid
   this warning.
```

This should appear in the description of the `Class` command, not here.  It's syntax related.

What are bad consequences of having a current `Class >` suddenly have a coercion in some future release?  How is this message really beneficial for users?

Reply:
> This should appear in the description of the Class command, not here. It's syntax related.

Good point, I updated the syntax of the command.

> What are bad consequences of having a current Class > suddenly have a coercion in some future release? How is this message really beneficial for users?

It's rather the other way around. Currently, this syntax is accepted but silently does nothing. So, if we make it effective in a future version, a user that would expect backward compatibility may get unpleasant errors, so better make this (very likely unused currently) syntax a hard error in at least one version before actually activating it.